source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -799,15 +800,21 @@ def fill_journeys(self, request_type, api_request, instance):
         journey_filter.final_filter_journeys(responses, instance, api_request)
         pb_resp = merge_responses(responses)
 
-        if 'ridesharing' in ridesharing_req['origin_mode']:
+        if 'ridesharing' in ridesharing_req['origin_mode'] and instance.ridesharing_services:
             period_extremity = PeriodExtremity(ridesharing_req['datetime'], ridesharing_req['clockwise'])
-            ridesharing_journey = build_ridesharing_crowfly_journey(instance,
-                                                                    ridesharing_req['origin'],
-                                                                    ridesharing_req['destination'],
-                                                                    period_extremity)
-            pb_resp.journeys.extend([ridesharing_journey])
-            if pb_resp.HasField(str('error')):
-                pb_resp.ClearField(str('error'))
+            try:
+                rs_journey, rs_tickets = build_ridesharing_crowfly_journey(instance,
+                                                                       ridesharing_req['origin'],
+                                                                       ridesharing_req['destination'],
+                                                                       period_extremity)
+                if rs_journey:
+                    pb_resp.journeys.extend([rs_journey])
+                    if rs_tickets:
+                        pb_resp.tickets.extend(rs_tickets)
+                    if pb_resp.HasField(str('error')):
+                        pb_resp.ClearField(str('error'))
+            except Exception as e:
+                logger.error('Error while retrieving ridesharing ads: {}'.format(e))
```

Comment:
I tested your branch with this request "http://canaltp.github.io/navitia-playground/play.html?request=http%3A%2F%2Flocalhost%3A5000%2Fv1%2Fcoverage%2Fsherbrooke%2Fjourneys%3Ffrom%3D-1.682103%253B48.109377%26to%3D-1.743929%253B48.020335%26datetime%3D20180110T070000%26datetime_represents%3Ddeparture%26first_section_mode%255B%255D%3Dridesharing%26"

some thing wrong at this line
i'm looking into it